test/e2e/tests/edit-gas-fee.spec.js

```diff
@@ -40,29 +40,28 @@ describe('Editing Confirm Transaction', function () {
 
         // update estimates to high
         await driver.clickElement('[data-testid="edit-gas-fee-button"]');
-        await driver.delay(regularDelayMs);
+        const timeElement = await driver.findElement({
+          text: '--',
+          tag: 'span',
+        });
+        await driver.waitForElementContainingValue(timeElement, 'sec');
```

Comment:
can we use `waitForSelector` here, instead of introducing a this new method `waitForElementContainingValue`?

Reply:
well seen. I've included some extra `waitForSelectors` in further steps, as it was intermittently failing otherwise